core/src/main/cpp/gdn/projection_mesh/rectangular_projection_mesh.cpp

```diff
@@ -0,0 +1,82 @@
+#include <gen/ArrayMesh.hpp>
+#include <gen/Mesh.hpp>
+#include <gen/QuadMesh.hpp>
+#include <gen/Shader.hpp>
+#include <gen/Shape.hpp>
+#include <utils.h>
+
+#include "projection_mesh_utils.h"
+#include "rectangular_projection_mesh.h"
+
+namespace gast {
+
+namespace {
+const float kCurvedScreenRadius = 6.0f;
+const size_t kCurvedScreenResolution = 20;
+const bool kDefaultCurveValue = false;
+const float kDefaultGradientHeightRatio = 0.0f;
+}  // namespace
+
+RectangularProjectionMesh::RectangularProjectionMesh(Vector2 mesh_size, bool is_curved) :
+        ProjectionMesh() {
```

Comment:
Let's explicitly specify the projection mesh type. Useful to avoid hard to spot bugs later on if we change the default projection type in `ProjectionMesh`.

Reply:
Done.